include/plotIt.h

```diff
@@ -57,6 +57,19 @@ namespace plotIt {
       return errors_type;
   }
 
+  enum Log {
```

Comment:
But there you're casting with (bool) which sould work with an enum class right? Anyway, it doesn't really matter much ;)